static/app/components/events/interfaces/searchBarActionV2.tsx

```diff
@@ -0,0 +1,119 @@
+import styled from '@emotion/styled';
+
+import DropdownButtonV2 from 'sentry/components/dropdownButtonV2';
+import CompactSelect from 'sentry/components/forms/compactSelect';
+import SearchBar from 'sentry/components/searchBar';
+import {t, tn} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+type FilterOption = React.ComponentProps<typeof CompactSelect>['options'][0];
+
+type Props = {
+  onChange: (value: string) => void;
+  placeholder: string;
+  query: string;
+  className?: string;
+  filterOptions?: FilterOption[];
+  filterSelections?: FilterOption[];
+  onFilterChange?: (options: FilterOption[]) => void;
+};
+
+function SearchBarAction({
```

Comment:
How many other places is the v1 used in?

Reply:
two more

Reply:
creating the PRs for them now